core/spec.md

```diff
@@ -1998,7 +2043,73 @@ response.
 Regardless of the mechanism used to update the capabilities, the Registry's
 `epoch` value MUST be incremented.
 
----
+In order for a client to discover the list of available values for each
+capability, an HTTP `GET` MAY be sent to the `/capabilities` API with the
+`?offered` query parameter and the response MUST adhere to the following:
+
+```yaml
+GET /capabilities?offered
+
+{
+  "STRING": [ ALLOWED-VALUES ] *
+}
+```
+
+Where:
+- `"STRING"` is the capability name.
+- `ALLOW_VALUES` is an array of the valid values for that capability. The
+  data type of each value MUST adhere to the data type associated with the
+  capability, with the exception of the `#` (range)  variants described below.
+
+For example:
+
+```yaml
+GET /capabilities?offered
+
+{
+  "enforcecompatibility": [ true, false ],
+  "flags": [ "doc", "epoch", "filter", "inline", "nodefaultversionid",
+    "nodefaultversionsticky", "noepoch", "noreadonly", "offered", "schema",
+    "setdefaultversionid", "specversion" ],
+  "mutable": [ "capabilities", "entities", "model" ],
+  "pagination": [ true, false ],
+  "schemas": [ "xRegistry-json/0.5" ],
+  "shortself": [ true, false ],
+  "specversions": [ "0.5" ]
+}
+```
+
+The array of values allows for some special cases:
+- `"*"` mean that all possible values of that data type are allowed.
+- Numeric capabilities MAY express ranges via `"#-#"` (min-max),
```

Comment:
I suspect that using `[x, y]`, `[x, y)`, `(x, y]` or `(x, y)` would be more flexible here, to allow for closed and open bounds.

Reply:
[x,] for x or higher

or ".." instead of ","

Reply:
Will address in https://github.com/xregistry/spec/issues/262